examples/peripheral_led/doc/index.md

```diff
@@ -23,4 +23,14 @@ Download the this example. Then test each function:
 * CTE responder in connection mode: Ref to [_Central CTE_](../../central_cte/doc/index.md)
     without `PRO_MODE`.
 
-* Proprietary connection-less AoA: Ref to [_Central CTE_](../../central_cte/doc/index.md) with `PRO_MODE`.
\ No newline at end of file
+* Proprietary connection-less AoA: Ref to [_Central CTE_](../../central_cte/doc/index.md) with `PRO_MODE`.
+
+
+##  RGB_LED Support
+
+At present, the program supports two LED light drivers, which can be adapted to two different LED lights by modifying the following macro definition：
+```
+#define LED_TLC59731    0
+#define LED_WS2881      1
+#define RGB_LED     LED_WS2881
+```
```

Comment:
At present, the program supports two LED light drivers (TLC59731 and WS2881), which can be adapted to two different LED lights by defining macro `RGB_LED` to following values：

```
#define LED_TLC59731    0
#define LED_WS2881      1
```